oak/server/rust/oak_runtime/src/node/mod.rs

```diff
@@ -30,9 +31,16 @@ mod wasm;
 
 /// Trait encapsulating execution of a Node or pseudo-Node.
 pub trait Node {
-    /// Execute the Node, using the provided `Runtime` reference and initial handle.  The method
+    /// Execute the Node, using the provided `Runtime` reference and initial handle. The method
     /// should continue execution until the Node terminates.
-    fn run(self: Box<Self>, runtime: RuntimeProxy, handle: oak_abi::Handle);
+    fn run(
+        self: Box<Self>,
+        runtime: RuntimeProxy,
+        handle: oak_abi::Handle,
+        // Object that receives a notification from the Runtime before it stopps the Node. This
+        // notification can be used to correctly shut down async routines executed by the Node.
```

Comment:
```suggestion
        // Object that receives a notification from the Runtime upon termination. This
        // notification can be used by the Node to gracefully shut down.
```

but also I don't think this comment shows up anywhere (e.g. in the IDE or in generated docs), it would be better to have it as part of the method documentation I think?

Reply:
Done